README.md

```diff
@@ -18,7 +18,7 @@ a customized adaptation.
 After the CI is green:
   * Update the version in the podspec to match the latest entry in the [CHANGELOG.md](CHANGELOG.md)
   * Add the CocoaPods tag
-    * Checkout the `main` branch.
+    * Checkout the `main` branch and ensure it is up to date.
```

Comment:
Good point! We may also express it as commands e.g.:
`git checkout master`
`git pull`

I guess it will be even more clear 😄.

Reply:
Never hurts– I'll add the relevant commands! :)